tool/processors/migration/linux/knownConfigKeys.go

```diff
@@ -81,5 +81,16 @@ func addLogConfig(logsConfig *config.Logs, filePath, section string, p *configpa
 			}
 		}
 	}
-	logsConfig.AddLogFile(logFilePath, logGroupName, logStreamName, timestampFormat, timezone, multiLineStartPattern, encoding)
+	retentionInDays, _ := p.Get(section, "retention_in_days")
+	retention := -1
+	if retentionInDays != "" {
+		if i, err := strconv.Atoi(retentionInDays); err == nil {
+			fmt.Printf("Retention is valid from config and value of %v", i)
+			retention = i
+		} else if err != nil {
```

Comment:
Do you need the else if here? Can't it just be an else since at this point you know that err is not nil?